csrankings-a.csv

```diff
@@ -2340,6 +2340,7 @@ Ashutosh Trivedi 0001,University of Colorado Boulder,https://ashut.bitbucket.io,
 Ashvin Goel,University of Toronto,http://www.eecg.toronto.edu/~ashvin,_AUSgBQAAAAJ
 Ashwin Ashok,Georgia State University,https://grid.cs.gsu.edu/~aashok/index.html,1SIKS28AAAAJ
 Ashwin Nayak 0001,University of Waterloo,http://www.math.uwaterloo.ca/~anayak/Site/Home.html,psqKefwAAAAJ
+Ashwin P. Dani,University of Connecticut,https://www.ee.uconn.edu/ashwin-dani/,b9cb6MEAAAAJ

```

Comment:
EE, not CS.

Reply:
Hi Emery,

Dr. Ashwin Dani is from ECE of UConn. He is the director of [Robotics and Controls Lab](https://rcl.engr.uconn.edu/), and have several publications in ICRA. To my understanding, he is qualified to be included in the website. Can you please include him?

Reply:
See below.